TreeStructure/GTNode.java

```diff
@@ -53,6 +61,7 @@ public void listChildren(){
         for(GTNode i : children){
         System.out.println(i.name);
         //System.out.println("List child operation went successful");
+        System.out.println(i.name + " (Created at: " + i.formattedTimestamp() + ")");

```

Comment:
I think you can just '...print(i)', since you already implemented the toString() function which is a string representation of the object.